core/ui/public/i18n/en/translation.json

```
@@ -424,7 +426,8 @@
         "set-metrics-configuration": "Set metrics configuration",
         "set-metrics-notifications": "Set metrics notifications",
         "get-metrics-id": "Get metrics ID",
-        "restart-module": "Restart instance"
+        "restart-module": "Restart instance",
```

The word "Instance" is deprecated. 
```suggestion
        "restart-module": "Restart application",
```

...or just Restart